### PR TITLE
test(frontend): TokenLogo component

### PR DESCRIPTION
**Pull request by AntonioVentilii**
# Motivation

Some tests for TokenLogo Component.

### CODE REVIEW

File: src/frontend/src/lib/components/tokens/TokenLogo.svelte [hunk 1]
```diff
@@ -42,6 +46,7 @@
 				src={badge.blackAndWhite ? networkIconBW : networkIcon}
 				alt={replacePlaceholders($i18n.core.alt.logo, { $name: networkName })}
 				{color}
+				testId={badgeTestId}
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/tokens/TokenLogo.svelte:49]: Is it expected that you use twice the same `testId` within the component?

Comment (AntonioVentilii) [reply src/frontend/src/lib/components/tokens/TokenLogo.svelte:49]: the component will render one or the other badge, it is exclusive; shall i maybe add a suffix/prefix, line `token-count-{badgeTestId}` and `network-{badgeTestId}` ?

Comment (peterpeterparker) [reply src/frontend/src/lib/components/tokens/TokenLogo.svelte:49]: Maybe? Somehow always thought `testId` are kind of unique.

Comment (AntonioVentilii) [reply src/frontend/src/lib/components/tokens/TokenLogo.svelte]: done! not really sure if should be prefix or suffix; have a look, WDYT? any preference?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**